go/test/endtoend/vtgate/errors_as_warnings/main_test.go

```diff
@@ -90,6 +90,7 @@ func TestMain(m *testing.M) {
 		}
 
 		// Start vtgate
+		clusterInstance.VtGateExtraArgs = []string{"-planner_version", "Gen4CompareV3"}
```

Comment:
instead of setting this manually everywhere, how about making this the default?

Reply:
I belive you mean default for test only.

Reply:
sequence tests will fail, because of the reason, it will be the next value always.

Reply:
Queries that use `next` (sequences) are ran against Gen4 only by design